game.py

```diff
@@ -241,8 +250,8 @@ def handle_collision(self):
         if self.move == 0:
             self.timer += 1
             if self.timer >= 125:
-                self.collision = 1
-        if self.collision == 1:
+                self.collision = True
+        if self.collision is True or self.coin_miss is True:
```

Comment:
This can be `if self.collision or self.coin_miss:`

Reply:
Fixed